src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/IHtmlHelper.cs

```diff
@@ -133,7 +135,7 @@ public interface IHtmlHelper
             string controllerName,
             object routeValues,
             FormMethod method,
-            bool suppressAntiforgery,
+            bool? suppressAntiforgery,
```

Comment:
BTW everyone using HTML helpers can always call `Html.AntiForgery()` to make the token addition unconditional.